pom.xml

```diff
@@ -22,28 +22,32 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!--Dependencies-->
-    <aspectj.version>1.9.20.1</aspectj.version>
+    <aspectj.version>1.9.21.1</aspectj.version>
```

Comment:
Suggestion: aspectjs should be moved in in the end along with log4j. In the next update prs it will be easier to found versions starting from raml so on..